src/Avalonia.Base/Input/PointerDeltaEventArgs.cs

```diff
@@ -7,7 +9,9 @@ public class PointerDeltaEventArgs : PointerEventArgs
     {
         public Vector Delta { get; set; }
 
-        internal PointerDeltaEventArgs(RoutedEvent routedEvent, object? source, 
+        [Unstable]
+        [Obsolete("This constructor may be removed in 12.0. Consider replacing it with RawPointerEventArgs or headless unit tests helpers.")]
```

Comment:
What's the reason to mention the headless platform here but not on `DragEventArgs`?

Reply:
We don't have anything in the headless for the drag events iirc

Reply:
Added DragDrop method to the IHeadlessWindow, updated comment.